src/Mvc/Mvc.ApiExplorer/src/EndpointMetadataApiDescriptionProvider.cs

```diff
@@ -243,15 +243,23 @@ private ApiDescription CreateApiDescription(RouteEndpoint routeEndpoint, string
                 responseType = typeof(void);
             }
 
-            var responseMetadata = endpointMetadata.GetOrderedMetadata<IApiResponseMetadataProvider>();
+            // For back-compat, we support attributes (which implement the IApiResponseMetadataProvider) interface
```

Comment:
back-compat seems like a strange term to use for a feature that's shipping for the first time